### PR TITLE
[BD] Fix runtime warnings

### PR DESCRIPTION
**Pull request by morenol**
## Description
Fix runtimewarnigs related to naive datetimes.

## Reviewers

- [ ] @andrey-canon
- [x] Ready for edX review
- [ ] @jmbowman

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4296](https://openedx.atlassian.net/browse/OSPR-4296) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: enterprise_data/api/v0/views.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def filter_active_enrollments(self, queryset):
         """
         return queryset.filter(
             has_passed=False,
-            course_end__gte=date.today(),
+            course_end__gte=timezone.now().replace(hour=0, minute=0, second=0, microsecond=0),
```
Comment (jmbowman) [inline enterprise_data/api/v0/views.py]: I think these two were buggy as written; they should have been using the current timestamp, not the day.  Seems more correct to just use `timezone.now()` without the `replace()` call.

File: enterprise_data/tests/test_views.py [hunk 2]
```diff
@@ -247,7 +248,12 @@ def test_get_queryset_returns_enrollments_with_passed_date_filter(self):
                 enterprise_user=enterprise,
                 enterprise_id=enterprise_id,
                 user_email='user{}@example.com'.format(index),
-                passed_timestamp=passed_date,
+                passed_timestamp=datetime(
+                    year=passed_date.year,
+                    month=passed_date.month,
+                    day=passed_date.day,
+                    tzinfo=pytz.utc,
+                ),
```
Comment (jmbowman) [inline enterprise_data/tests/test_views.py]: Might be simpler to just replace `date_today = date.today()` above with `date_today = timezone.now()`.  You'd also need to adjust an assertion below.

**Comment by morenol** [comment]
The only warning after this change is: 

 UserWarning: EdxRestApiClient is deprecated. Use OAuthAPIClient instead.
    warnings.warn('EdxRestApiClient is deprecated. Use OAuthAPIClient instead.')

**Review by jmbowman — APPROVED** [review]

**Pull request merged**